core/snippets/snippets/markdown.snippet

```diff
@@ -4,5 +4,48 @@ language: markdown
 name: link
 phrase: link
 -
-[$1]($0)
+[$0]($1)
```

Comment:
You seem to have inverted the order of the snippet holes for the link snippet. Was there a reason for this?

Reply:
Now I have seen the other comment. I think the latter is the better solution, so I have updated it to that. However, this does change the stops slightly for that snippet. Is that likely to be a problem, do you think?

Reply:
I think the argument that it would be convenient to have a quick way to exit the snippet structure could be applied to many snippets. I personally think it would be better to have a command that works in the general case for that instead of modifying individual snippets. 
I can discuss this with whoever shows up at the next backlog session (tomorrow).
I doubt the maintainers would want to change the existing link snippet given that it was part of the migration, and we do not want to create breaking changes too frequently. I would leave it for now though and wait for their input.

Reply:
That makes sense and would make it inconsistent with all the other snippets. I will revert it

Reply:
Feedback from community session. We agree that we want to keep the current snippet behavior.